CONTRIBUTING.md

```diff
@@ -444,17 +444,23 @@ When choosing `minor` or `major`, make sure your PR targets the `next` branch in
 
 The description of the changeset should follow the these guidelines:
 
-- Our changesets should be about _user-facing_ changes. Internal changes don't
-  need changesets.
-- Use the past tense when describing what you did, e.g. "Added new feature", "Fixed edge case".
-- Use the present tense when describing Biome behavior, e.g. "Biome now supports ...".
+- Our changesets should be about **user-facing changes**. Internal changes don't
+  need changesets. For example, if you refactored some code but the user-facing
+  behavior didn't change, you don't need a changeset.
+- **Be concise and clear.** Changesets are not documentation and they are not test cases.
+  They should give a **quick overview** of what changed, allowing the reader to dig deeper if they want to. A good changeset is usually between 1 and 3 sentences long. **Longer changesets indicate to the user that they should pay more attention to the change, but that is not always the case.** Consider the impact of your change on the user when writing the changeset. The key is to provide just enough information for the user to understand the change without overwhelming them with details.
+  - For a *new* lint rule, show an example of an invalid case in a code block. If it _really_ helps demonstrate the rule, you can also show a valid case.
+  - For a *change in an existing rule*, clearly demonstrate what is now considered invalid that wasn't before, or vice versa. If it helps communicate the change greatly, show both invalid and valid cases.
+  - For a *formatter change*, show an example of the formatting change using a `diff` code block.
+  - For parser changes, a brief inline example of the what can now be parsed that couldn't before (or vice versa) is usually sufficient. If it would be formatted better on multiple lines, use a code block.
+- **Use past tense when describing what you did**, e.g. "Added new feature", "Fixed edge case".
+- **Use the present tense when describing Biome behavior**, e.g. "Biome now supports ...".
 - If you fixed a bug, please start with a link to the issue, e.g. "Fixed [#4444](https://github.com/biomejs/biome/issues/4444): ...".
 - If you reference a rule, please add the link to the rule on the website, e.g. "Added the rule [`useAwesomeThing`](https://biomejs.dev/linter/rules/use-awesome-thing/)" (even if the website isn't updated yet, the URL is pretty predictable...).
```

Comment:
```suggestion
- If you reference a rule, please add the link to the rule on the website, e.g. "Added the rule [`useAwesomeThing`](https://biomejs.dev/linter/rules/use-awesome-thing/)" (even if the website isn't updated at the time of writing, it will once the PR is merged).
```